lib/workers/repository/error-config.spec.ts

```diff
@@ -24,9 +30,16 @@ describe('workers/repository/error-config', () => {
       const error = new Error(CONFIG_VALIDATION);
```

Comment:
we should create a custom error class for this, so the properties can easily passt as constructor params and we can directly throw it.

should be done in separate PR of cause